controllers/capabilities/developeruser_threescale_reconciler.go

```diff
@@ -57,11 +60,15 @@ func (s *DeveloperUserThreescaleReconciler) Reconcile() (*threescaleapi.Develope
 
 	if devUser == nil {
 		s.logger.V(1).Info("DeveloperUser does not exist", "username", s.userCR.Spec.Username)
-		// developer user has to be created in 3scale
+		// The DeveloperAccount doesn't exist yet and must be created in 3scale
```

Comment:
DeveloperUser?

Reply:
Good catch, I pushed changes to fix the typos